docs/core/runtime-config/garbage-collector.md

```diff
@@ -824,3 +824,64 @@ For more information about the first 3 settings, see the [Middle ground between
 | **runtimeconfig.json**   | `System.GC.DynamicAdaptationMode` | `1` - enabled<br/> `0` - disabled | .NET 8 |
 
 [!INCLUDE [runtimehostconfigurationoption](includes/runtimehostconfigurationoption.md)]
+
+#### Target TCP
+
+DATAS uses the Throughput Cost Percentage (TCP) as the memory cost measurement on throughput. It takes both GC pauses and how much allocations have to wait into consideration. Usually the former  dominates so you can use the "% pause time in GC" to approximate this cost. There are 2 transient cases where the latter dominates -
+
+- When a process starts, DATAS always starts with one heap so if there are enough threads allocating they experience waits.
+- When the workload changes from lighter to heavier, eg, when peak hours start, since it takes a few GCs before the number of heaps gets increased, allocating threads may experience waits during that period of time.
```

Comment:
```suggestion
DATAS uses the Throughput Cost Percentage (TCP) as the memory cost measurement on throughput. It takes into consideration both GC pauses and how much allocations have to wait. Usually, GC pauses dominates, so you can use the "% pause time in GC" to approximate this cost. There are two transient cases where the allocation wait time dominates:

- When a process starts, DATAS always starts with one heap. So if there are enough threads allocating, they experience waits.
- When the workload changes from lighter to heavier, for example, when peak hours start, allocating threads might experience waits during that period of time. That's because it takes a few GCs before the number of heaps gets increased.
```

Reply:
I'll change `There are two transient cases where the allocation wait time dominates:` to There are two transient cases where the allocation wait time can dominate" since it's not a sure thing.